kafka-ui-react-app/src/components/App.tsx

```diff
@@ -39,21 +43,85 @@ const queryClient = new QueryClient({
     },
   },
 });
+const options = [
+  {
+    label: (
+      <>
+        <AutoIcon /> Auto theme
+      </>
+    ),
+    value: 'Auto theme',
```

Comment:
since value labels are not visible to the user, lets make them lowercase and with `_` , `auto_theme`